### ISSUE TITLE
No Data for Sign-Ins means something different than No Data for the Canvas measures

### ISSUE THREAD

**Issue by henrylyle** [uw-it-aca/retention-dashboard#164]
For Sign-Ins No data indicates that the student has not signed in to any UW system during the specific time range. For Canvas, it indicates Canvas use is not a course requirement. We might wanna change the terminology as it has caused some confusion. Maybe "No Sign-Ins" instead. Also, a user suggested that we include "No Data" for Sign-Ins when selecting the "low" filter. For instance, if lyle3 hasn't logged in at all, he is coded as No Data and will not appear in the "low" filter.

**Comment by bexar** [comment]
Agreed. Let's change terminology to "No Sign-Ins" and include "No Sign-Ins" when selecting the "Low" filter.
